code/game/gamemodes/cult/cult_items.dm

```diff
@@ -10,6 +10,7 @@
 	drop_sound = 'sound/items/drop/sword.ogg'
 	pickup_sound = 'sound/items/pickup/sword.ogg'
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	origin_tech = list(TECH_ARCANE = 3)

```

Comment:
As these are commonly acquireable outside of cult rounds, it's probably safer to have arcane at max be 2. A bio of 4 or 5 is not unheard of, though, as the object is suspiciously animate.